src/SlmLocale/Strategy/UriPathStrategy.php

```diff
@@ -124,7 +129,7 @@ public function found(LocaleEvent $event)
         }
 
         $locale = $event->getLocale();
-        if (null === $locale) {
+        if (null === $locale || $locale === $this->default) {
```

Comment:
this can be BC break I guess if we want this we will have to introduce one more option to enable this functionality. Because this will change the URL for existing users. At least maybe add `$this->default !== null`.